src/main/target/SITL/target.c

```diff
@@ -474,15 +430,13 @@ void FLASH_Unlock(void) {
         fprintf(stderr, "[FLASH_Unlock] eepromFd != NULL\n");
         return;
     }
-
     // open or create
-    eepromFd = fopen(EEPROM_FILENAME,"r+");
+    eepromFd = fopen(EEPROM_FILENAME, "r+");
```

Comment:
Codacy found an issue: [Check when opening files - can an attacker redirect it (via symlinks), force the opening of special file type (e.g., device files), move things around to create a race condition, control its ancestors, or change its contents? (CWE-362).  ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=6326343)